website/replay.php

```diff
@@ -61,12 +61,17 @@ function poll_as_replay() {
 var g_remote_poller;
 var g_recorder;
 
-var g_replay_count;
-var g_replay_rate;
+var g_replay_duration = <?php echo read_raceinfo('replay-skipback', '4'); ?>;
+var g_replay_count = <?php echo read_raceinfo('replay-num-showings', '2'); ?>;
```

Comment:
g_replay_count and g_replay_rate are set by the replay message; initializing them here doesn't really have any effect (but misleadingly gives the impression that it does).